src/renderer/commands/createDrawObject.ts

```diff
@@ -8,36 +9,36 @@ import REGL = require('regl');
 
 // Uniforms are the same for all vertices.
 interface Uniforms {
-    projection: REGL.Mat4;
-    view: REGL.Mat4;
-    model: REGL.Mat4;
+    projection: mat4;
+    view: mat4;
+    model: mat4;
     numLights: number;
-    ambientLight: REGL.Vec3;
-    lightPositions: REGL.Vec3[];
-    lightColors: REGL.Vec3[];
-    lightIntensities: number[];
+    ambientLight: vec3;
     isShadeless: boolean;
 }
 
 // Attributes are per vertex.
 interface Attributes {
-    position: REGL.Vec3;
-    normal: REGL.Vec3;
-    color: REGL.Vec3;
+    position: Float32Array;
+    normal: Float32Array;
+    color: Float32Array;
 }
 
 /**
  * All the information needed to be able to draw an object to the screen
  */
 export interface DrawObjectProps {
-    model: REGL.Mat4;
-    cameraTransform: REGL.Mat4;
-    projectionMatrix: REGL.Mat4;
-    positions: REGL.Vec3[];
-    normals: REGL.Vec3[];
-    colors: REGL.Vec3[];
-    indices: number[];
+    model: mat4;
+    cameraTransform: mat4;
+    projectionMatrix: mat4;
+    positions: Float32Array;
+    normals: Float32Array;
+    colors: Float32Array;
+    indices: Int16Array;
+    numLights: number;
```

Comment:
Why are `numLights` and `ambientLight` here now as well?

Reply:
They were always being passed in, but because typechecking for regl was entirely broken before, we just didnt notice that we forgot to add them to the interface